website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -132,29 +79,28 @@ Note that `toggleText` is still required–it provides the `aria-label` for the
 </Hds::Dropdown>
 ```
 
-### With a loading "interactive" item
+#### ToggleIcon as user menu
 
-There may be use cases when it’s necessary to put an item in a "loading" state while the app performs some operations (e.g., checking asynchronously the user’s permission to execute a certain operation, once the toggle has been clicked).
-
-In that case the argument `@isLoading={{true}}` can be passed to the item: this will show a "loading" icon (even if an argument `@icon` is provided) and set the item as non-interactive until the value of `@isLoading` is set to `false` again.
+`@text` is still required, because it supplies the `aria-label` for ToggleIcon.
 
 ```handlebars
 <Hds::Dropdown as |dd|>
-  <dd.ToggleIcon @icon="more-horizontal" @text="Overflow Options" @hasChevron={{false}} />
-  <dd.Interactive @route="components" @isLoading={{true}} @text="Edit cluster" @color="action" @icon="edit" />
+  <dd.ToggleIcon @icon="user" @text="user menu" />
+  <dd.Title @text="Signed In" />
+  <dd.Description @text="email@domain.com" />
+  <dd.Separator />
+  <dd.Interactive @route="components" @text="Settings and Preferences" />
   <dd.Interactive @route="components" @text="Delete" @color="critical" @icon="trash" />
 </Hds::Dropdown>
 ```
 
-### ToggleIcon as user menu
-
-In this example, we have a user icon with a title, description, separator, and links.
+#### ToggleIcon with other icons
 
-Note that `toggleText` is still required, because it supplies the `aria-label` for the toggle button.
+Pass any [Helios icon](/foundations/icons/library) name to `@icon` to change the icon used in ToggleIcon. `@text` is still required, because it supplies the `aria-label` for ToggleIcon.
```

Comment:
```suggestion
Pass any [Helios icon](/icons/library) name to `@icon` to change the icon used in ToggleIcon. `@text` is still required, because it supplies the `aria-label` for ToggleIcon.
```
If I missed some of these in other PRs, don't worry about it for now, I have a ticket open to check all of the links.